packages/pocket-ic/src/common/rest.rs

```diff
@@ -356,6 +356,23 @@ impl From<Principal> for RawNodeId {
     }
 }
 
+#[derive(Debug, Clone, Eq, PartialEq, Serialize, Deserialize, JsonSchema, Default)]
+pub struct TickConfigs {
+    pub blockmakers: Option<BlockMakerConfigs>,
+}
+
+#[derive(Debug, Clone, Eq, PartialEq, Serialize, Deserialize, JsonSchema)]
+pub struct BlockMakerConfigs {
```

Comment:
```suggestion
pub struct BlockmakerConfigs {
```
to be consistent with `RawSubnetBlockmakers`